### DISCUSSION TITLE
Looking forward to adding the files tags function

### DISCUSSION THREAD

**Discussion by elicec** [doublecmd/doublecmd#1677] [Suggestions]
like macos

**Comment by ags1234** [comment]
Tag by color? That would be excellent!
But how do you store that on the disk? In alternate data streams in NTFS? And the rest of the multitude of file systems?
Or DC builds it's own database of files and their attributes?

**Reply by elicec** [reply]
I think "DC builds it's own database of files and their attributes?" is OK.
Tags can help people classify files

**Comment by Skif-off** [comment]
Maybe talking about extended attributes? `setfattr`, `getfattr` and so on.
A WDX plugin was written to display them ([getfattrcustomwdx.lua](https://github.com/j2969719/doublecmd-plugins/blob/master/plugins/wdx/scripts/getfattrcustomwdx.lua)), i.e. for column sets, fooltips and file search. [SetFAttrCustom.lua](https://github.com/j2969719/doublecmd-plugins/blob/master/scripts/lua/SetFAttrCustom.lua) allows to change such attributes.

_Edit:_ I forgot about [Tags](https://github.com/j2969719/doublecmd-plugins/tree/master/scripts/lua/tags) and [Marker](https://github.com/j2969719/doublecmd-plugins/tree/master/scripts/lua/marker).

_Edit:_ Links have been updated.

**Reply by elicec** [reply]
Okay, thanks. I'll try this. 
This looks like exactly what I want.

**Reply by reregaga** [reply]
Thank you. Correct [link](https://github.com/j2969719/doublecmd-plugins/tree/24.02.27/utils) for Tags & Marker.

**Reply by Skif-off** [reply]
@reregaga, thanks, fixed.
